docs/middleware/BLSRegistryCoordinatorWithIndices.md

```diff
@@ -0,0 +1,80 @@
+# BLSRegistryCoordinatorWithIndices
+
+This contract is deployed for every AVS and serves as the main entrypoint for operators to register and deregister from the AVS. In addition, it is where the AVS defines its operator churn parameters.
```

Comment:
If the architecture is getting explored / highlighted in any one of these contract-specific documentation files, I think this is the place.

Reply:
put it in the README instead and an index for the folder